variables.tf

```diff
@@ -132,6 +132,18 @@ variable "artifact_location" {
   description = "Location of artifact. Applies only for artifact of type S3"
 }
 
+variable "secondary_artifact_location" {
+  type        = string
+  default     = ""
```

Comment:
Can you set the default to `null`? That's typical for default string values.

Reply:
Will do. There are a few other places in the module where empty strings are used. I can address that as well, but was thinking of doing that in a separate PR, unrelated to this one. Does that sounds okay?

Reply:
@jhosteny good by me -- you don't need to fix other stuff that likely just slipped through 👍